examples/index.md

```diff
@@ -290,33 +247,40 @@ Content-Type: application/vnd.api+json
 {
   "errors": [
     {
-      "title": "Invalid Attribute",
       "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
       "detail": "First name must contain at least three characters."
     },
     {
-      "title": "Invalid Attribute",
       "source": { "pointer": "data/attributes/first-name" },
+      "title": "Invalid Attribute",
```

Comment:
This is just me neurotically reordering the properties for consistency. Throughout these examples the order is now: status, code, source, title, detail.